libraries/AP_InertialSensor/AP_InertialSensor.h

```diff
@@ -268,6 +273,74 @@ class AP_InertialSensor : AP_AccelCal_Client
     // return time in microseconds of last update() call
     uint32_t get_last_update_usec(void) const { return _last_update_usec; }
 
+    enum IMU_SENSOR_TYPE {
+        IMU_SENSOR_TYPE_ACCEL,
```

Comment:
minor point, given this forms part of the log data (so part of a protocol), I think giving the specific values with =0 and =1 is worthwhile in the enum

Reply:
Done.